src/fermilib/transforms/_bksf.py

```diff
@@ -0,0 +1,449 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""bravyi_kitaev_fast transform on fermionic operators."""
+from __future__ import absolute_import
+
+import networkx
+import numpy
+from projectq.ops import QubitOperator
+
+from fermilib.ops import InteractionOperator
+from fermilib.utils import count_qubits
+
+
+def bravyi_kitaev_fast(operator):
+    """ Find the Pauli-representation of InteractionOperator for Bravyi-Kitaev
+    Super fast (BKSF) algorithm. Pauli-representation of general
+    FermionOperator is not possible in BKSF. Also, the InteractionOperator
+    given as input must be Hermitian. In future we might provide a
+    transformation for a restricted set of fermion operator.
+
+    Args:
+        operator: Interaction Operator.
+
+    Returns:
+        transformed_operator: An instance of the QubitOperator class.
+
+    """
+    if isinstance(operator, InteractionOperator):
+        return bravyi_kitaev_fast_interaction_op(operator)
+    else:
+        TypeError("operator must be an InteractionOperator.")
```

Comment:
Yeah, I looked at the test results and I will add more tests or change present tests to include all the code.
Will fix this in the next commit.